system/gameModes/5e/load.lst

```diff
@@ -1,42 +1,43 @@
-#SIZEMULT:Fine|0.125
-#SIZEMULT:Diminutive|0.25
-SIZEMULT:Tiny|0.5
-SIZEMULT:Small|0.75
-SIZEMULT:Large|2
-SIZEMULT:Huge|4
-SIZEMULT:Gargantuan|8
-#SIZEMULT:Colossal|16
+#SIZEMULT:Fine|1
+#SIZEMULT:Diminutive|2
+SIZEMULT:Tiny|4
+SIZEMULT:Small|6
+SIZEMULT:Medium|8
+SIZEMULT:Large|16
+SIZEMULT:Huge|32
+SIZEMULT:Gargantuan|64
+#SIZEMULT:Colossal|128
 LOAD:0|0
-LOAD:1|5
-LOAD:2|10
-LOAD:3|15
-LOAD:4|20
-LOAD:5|25
-LOAD:6|30
-LOAD:7|35
-LOAD:8|40
-LOAD:9|45
-LOAD:10|50
-LOAD:11|55
-LOAD:12|60
-LOAD:13|65
-LOAD:14|70
-LOAD:15|80
-LOAD:16|85
-LOAD:17|90
-LOAD:18|100
-LOAD:19|105
-LOAD:20|110
-LOAD:21|115
-LOAD:22|120
-LOAD:23|125
-LOAD:24|130
-LOAD:25|135
-LOAD:26|140
-LOAD:27|145
-LOAD:28|150
-LOAD:29|155
-LOAD:30|160
+LOAD:1|0.625
+LOAD:2|1.25
+LOAD:3|1.875
+LOAD:4|2.5
+LOAD:5|3.125
+LOAD:6|3.75
+LOAD:7|4.375
+LOAD:8|5
+LOAD:9|5.625
+LOAD:10|6.25
+LOAD:11|6.875
+LOAD:12|7.5
+LOAD:13|8.125
+LOAD:14|8.75
+LOAD:15|9.375
+LOAD:16|10
+LOAD:17|10.625
+LOAD:18|11.25
+LOAD:19|11.875
+LOAD:20|12.5
+LOAD:21|13.125
+LOAD:22|13.75
+LOAD:23|14.375
+LOAD:24|15
+LOAD:25|15.625
+LOAD:26|16.25
+LOAD:27|16.875
+LOAD:28|17.5
+LOAD:29|18.125
+LOAD:30|18.75
 LOADMULT:4
```

Comment:
Aspect of the bear now gives bonus to load multipliers. I had to change the load.lst in order to do this correctly for small and smaller creatures due to the fact that you can't give fractional bonuses. The base weight loads are now adjusted so that a fine creature's SIZEMULT is 1. The load for each point of strength is adjusted to be 5/8 of what it was. The loads for each creature should be the same but this will need to be further verified with things such as item weight. The line that gives these bonuses will have to be changed if/when fine and diminutive size categories become available.